src/config.js

```diff
@@ -0,0 +1,39 @@
+/*
+Configuration Settings
+----------------------
+The config settings change depending on which environment the app is running in.
+By default, this is the development environment, but this can be changed by:
+- An env query string, e.g. localhost:3998?env=production, or
+- The NODE_ENV environment variable on the system running the app.
```

Comment:
This isn’t true for Webpack 4 or 5. `NODE_ENV` is the Webpack build mode, which can be one of `development`, `production` or `none`. `staging` isn’t a valid value. 

https://webpack.js.org/configuration/mode/